pbpy/pbgit.py

```diff
@@ -47,77 +47,77 @@ def get_lfs_version():
 
 
 def set_tracking_information(upstream_branch_name: str):
-    subprocess.call(["git", "branch", "--set-upstream-to=origin/" +
-                     upstream_branch_name, upstream_branch_name])
+    subprocess.run(["git", "branch", f"--set-upstream-to=origin/{upstream_branch_name}", upstream_branch_name])
 
 
 def stash_pop():
     pblog.info("Trying to pop stash...")
 
-    output = subprocess.getoutput(["git", "stash", "pop"])
-    pblog.info(str(output))
+    run = pbtools.run_with_output(["git", "stash", "pop"])
+    pblog.info(run.stdout)
+    pblog.error(run.stderr)
 
-    lower_case_output = str(output).lower()
+    output = f"{run.stdout}\n{run.stderr}"
+    lower_case_output = output.lower()
 
     if "auto-merging" in lower_case_output and "conflict" in lower_case_output and "should have been pointers" in lower_case_output:
-        pbtools.error_state("""Git stash pop is failed. Some of your stashed local changes would be overwritten by incoming changes.
-        Request help on #tech-support to resolve conflicts, and  please do not run StartProject.bat until issue is solved.""", True)
+        pbtools.error_state("""git stash pop failed. Some of your stashed local changes would be overwritten by incoming changes.
+        Request help in #tech-support to resolve conflicts, and please do not run StartProject.bat until the issue is resolved.""", True)
     elif "dropped refs" in lower_case_output:
         return
     elif "no stash entries found" in lower_case_output:
         return
     else:
-        pbtools.error_state("""Git stash pop is failed due to unknown error. Request help on #tech-support to resolve possible conflicts, 
-        and  please do not run StartProject.bat until issue is solved.""", True)
+        pbtools.error_state("""git stash pop failed due to an unknown error. Request help in #tech-support to resolve possible conflicts, 
+        and please do not run StartProject.bat until the issue is resolved.""", True)
 
 
 def check_remote_connection():
```

Comment:
After the recent changes to that function, pbsync prints a huge output to the console:

```
From https://github.com/ProjectBorealisTeam/pb.git
ba1d33f447f2070163674f1b5b03df7fbf5f4cc5        refs/heads/code/ai/combine/grenade-avoiding
e7f132375597a9dcba7f5d965585a8eb217466d5        refs/heads/code/ai/combine/grenade-throwing
5a1a77283b3cb06ee42c456078116961bb86d442        refs/heads/code/event-states
73e7920088b5c4d60afec688ffbbc7a0d92cf5be        refs/heads/code/fire-effects
9a09036b45c945aa4d61728f3eac0e2a3e16830c        refs/heads/code/first-pass-on-weapon-data-assets
5433a490932e6b28609de525865ed79b32374c96        refs/heads/code/headcrab-burrow
231c9aead1ca3c250bc04644738adbeccf5a4db6        refs/heads/code/pbsync-updates
a0b859b178e00657deff2c0f9583e9537503f8fc        refs/heads/code/performance-tweaks
e6be17a66702774f929eae46a4afd85e8a4e7d38        refs/heads/code/rope
d264f2ef5613db15c5246f83df9ec13e0ed4cb6a        refs/heads/code/volume-fog
bf99f9572dfb54d4bb17fd4c1601832392a58c7a        refs/heads/master
f0984bc1ea9672ef9d45cff8b8b14ed9ebd76ce5        refs/heads/promoted
b9097a2e6f4a61d62db71c3133741ab8349e21ae        refs/heads/release
b430660764fac4ca1d19fb544a6f45585ca64a10        refs/heads/stable
681e3f2e6251c297bcf4b950430b99b17ca106f7        refs/heads/trunk
```

Reply:
Should be fixed now, can you test again?